docs/DeveloperGuide.md

```diff
@@ -743,38 +834,96 @@ be able to view the applicant from the list.
     * 1a1. TrackAScholar displays an error message prompting correct input format. <br>
       Use case resumes at step 1.
 
-* 1b. TrackAScholar has no applicants stored.
+* 1b. Input parameters is invalid.
 
-    * 1a1. TrackAScholar shows an empty applicant list.
+    * 1b1. TrackAScholar displays an error message showing parameter requirements. <br>
+      Use case resumes at step 1.
 
 --------------------------------------------------------------------------------------------------------------------
 
-**Use case: UC05 - Filter applicants by status**
+**Use case: UC09 - Filter applicants by application status**
 
 **MSS**
 
 1. User requests a filtered list based on a specific application status.
-2. TrackAScholar shows the number of applicants by status.
-3. TrackAScholar shows the percentage of applicants by status.
-4. TrackAScholar shows the filtered list of applicants with the specified applicant status.
+2. TrackAScholar shows the filtered list of applicants with the specified applicant status.
 
     Use case ends.
 
 **Extensions**
 
-* 1a. User specifies more than one applicant status during request.
+* 1a. Input format is wrong.
 
-    * Step 4 repeats as many times as there are requests.
+    * 1a1. TrackAScholar displays an error message prompting correct input format. <br>
+      Use case resumes at step 1.
 
-* 1b. TrackAScholar detects an error in the entered applicant status/detects no applicant status input.
+* 1b. Input parameters is invalid.
 
-    * 1b1. TrackAScholar requests for the correct applicant status.
-    * 1b2. User enters new applicant status.
-    * Steps 1b1-1b2 are repeated until the applicant status entered matches a valid existing applicant status.
-    * Use case resumes from step 2.
+    * 1b1. TrackAScholar displays an error message showing parameter requirements. <br>
+      Use case resumes at step 1.
+
+--------------------------------------------------------------------------------------------------------------------
 
+**Use case: UC10 - Sorts all applicants by name, scholarship or application status**
 
-*{More to be added}*
+**MSS**
+
+1. User requests a sorted list sorted by name, scholarship or application status.
+2. TrackAScholar shows the sorted list of applicants with the specified input parameter. <br>
+   Use case ends.
+
+**Extensions**
+
+* 1a. Input format is wrong.
+
+    * 1a1. TrackAScholar displays an error message prompting correct input format. <br>
+      Use case resumes at step 1.
+
+* 1b. Input parameters is invalid.
+
+    * 1b1. TrackAScholar displays an error message showing parameter requirements. <br>
+      Use case resumes at step 1.
+
+--------------------------------------------------------------------------------------------------------------------
+
+**Use case: UC11 - Imports a json file into TrackAScholar**
+
+**Guarantees:**
+* After merging, no two applicants will have the same name.
```

Comment:
no two applicants ***in TrackAScholar***

Reply:
noted, added improvement.